wpimath/src/main/java/edu/wpi/first/math/MathUtil.java

```diff
@@ -145,4 +145,41 @@ public static double angleModulus(double angleRadians) {
   public static double interpolate(double startValue, double endValue, double t) {
     return startValue + (endValue - startValue) * MathUtil.clamp(t, 0, 1);
   }
+
+  /**
+   * Checks if the given value matches an expected value within a certain tolerance.
+   *
+   * @param expected The expected value
+   * @param actual The actual value
+   * @param tolerance The allowed difference between the actual and the expected value
+   * @return Whether or not the actual value is within the allowed tolerance
+   */
+  public static boolean isNear(double expected, double actual, double tolerance) {
+    if (tolerance < 0) {
+      throw new IllegalArgumentException("Tolerance must be a non-negative number!");
+    }
+    return Math.abs(expected - actual) < tolerance;
+  }
+
+  /**
+   * Checks if the given value matches an expected value within a certain tolerance. Supports
+   * continuous input for cases like absolute encoders
+   *
+   * @param expected The expected value
+   * @param actual The actual value
+   * @param tolerance The allowed difference between the actual and the expected value
+   * @param min The minimum value expected from the input
+   * @param max The maximum value expected from the input
```

Comment:
I personally find "____ value expected from the input" confusing in this context since there's another interpretation of expected values, some kind of wording more clearly related to wrapping would help.

Reply:
Maybe "Smallest value before wrapping around to the largest value" and "Largest value before wrapping around to the smallest value"? Will users know what "wrapping around" means?
If necessary, I can also put in an explanation like "If the given value is less than min, the difference between min and the given value will be subtracted from max until the given value is above min."

Reply:
Avoid explaining the exact algorithm in the high-level docs.

> Maybe "Smallest value before wrapping around to the largest value" and "Largest value before wrapping around to the smallest value"? Will users know what "wrapping around" means?

I'd do this but also give an example with some concrete numbers.